ag/ag.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.19.4
+// 	protoc        v3.17.3
```

Comment:
Please update your protobuf installation to avoid potential issues due to older version.

Reply:
Done. The latest version is 3.20.0, should we move to it instead?